core/src/main/java/com/oath/oak/MemoryManager.java

```diff
@@ -8,6 +8,8 @@
 
 
 
+import com.oath.oak.NativeAllocator.OakNativeMemoryAllocator;
+
 import java.nio.ByteBuffer;
 
 
```

Comment:
Do oak ever use the methods other than the last 3 that are used inside Chunk

Reply:
Maybe I am not understanding the comment, but yes, all MemoryManager methods are in use...